src/main/java/org/shanerx/tradeshop/commands/commandrunners/ShopFindSubCommand.java

```diff
@@ -126,8 +126,9 @@ public void run() {
                     if (desiredCost.size() == 0)
                         desiredCost.put(0, null);
 
-                    int finalDesiredRange = desiredRange > 0 ? desiredRange : Setting.DEFAULT_FIND_RANGE.getInt();
+                    final int finalDesiredRange = Math.min(desiredRange, Setting.MAX_FIND_RANGE.getInt());
```

Comment:
These are not the same, a negative range will be passed on by the Min method where the previous method would only return the desired range if it was greater than zero. 

This may work if you min(abs(desiredRange), defaultRange)

Reply:
This is intentional. Negative values in config mean no search, which will be handled explicitly in the function which compares distances (I think findProximityShops or something)

Reply:
Should probably document that though